bindata/network/ovn-kubernetes/common/80-ipsec-worker-extensions.yaml

```diff
@@ -0,0 +1,29 @@
+{{if .IPsecMachineConfigEnable}}
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 80-ipsec-worker-extensions
```

Comment:
IMO it would be good to go with two different files because these rendered as two different MC objects. This is more readable than creating two objects via programmatically.